group_vars/customer/eugene/ci/secrets/nextcloud.vault.yml

```diff
@@ -0,0 +1,15 @@
+# customer: jctbue
+# env_type: production
+
+# postgresql
+
+DB_USER: nextcloud_user
+DB_PASSWORD: pass
+
+NEXTCLOUD_SECRETS: 
+  POSTGRES_USER: nextcloud_user
+  POSTGRES_PASSWORD: pass
+  NEXTCLOUD_ADMIN_PASSWORD: password
+  NEXTCLOUD_PASSWORD_SALT: pjjs7qzjdCR2A7dpd6CvFdcn3DAcaZ
+  NEXTCLOUD_SECRET: sSNYeHyF8tXRitCbprJ7kMLnym9SMP
+  INSTANCE_ID: MGb3cQ5Y7y
```

Comment:
Should be encrypted with `ansible-vault`.

Reply:
Oups